litmus-portal/frontend/src/pages/LoginPage/index.tsx

```diff
@@ -79,7 +79,7 @@ const LoginPage = () => {
                 validationError={validateStartEmptySpacing(authData.username)}
                 data-cy="inputEmail"
                 required
-                handleChange={(e) =>
+                handleChange={(e: { target: { value: any } }) =>
```

Comment:
Remove this, you are type casting the event to be of type any whereas it is automatically infering the type to `React.ChangeEvent<HTMLInputElement>`